oqsprov/oqs_hyb_kem.c

```diff
@@ -0,0 +1,323 @@
+// SPDX-License-Identifier: Apache-2.0 AND MIT
```

Comment:
Key question: Why a separate file? Conceptually, a hybrid KEM provider also contains all the base logic for an OQS-only KEM provider, no?

Reply:
Now merged in a single file, see comment above.